### PR TITLE
pushing the code files

### PR DESCRIPTION
**Pull request by mihirotron**

### CODE REVIEW

File: src/CrowdDataDashboard.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        .live-updates .updates-container {
+            max-width: 100%; /* Ensure the container spans the full width of the page */
+            height: auto; /* Automatically adjust the height based on content */
+            display: block; /* Ensure the container is treated as a block element */
+            margin: 0 auto; /* Center the container horizontally */
+        }
+    </style>
+</head>
+<body>
+    <header>
```
Comment (doubleedesign) [inline src/CrowdDataDashboard.html:18]: Everything in the `body` needs to be broken up into React components.

File: src/ResourcesAndSupport.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+</head>
+<body>
+    <header>
+		<img src="Logo.svg" alt="Crowd Management Platform Logo">
+		<div>
+			<h1>Resources and Support</h1>
+		</div>
+    </header>
+    <main>
+        <section class="guidelines">
```
Comment (doubleedesign) [inline src/ResourcesAndSupport.html:18]: Likewise for this file - React components. There should not be any `.html` files added.

File: src/assets/styles.css [hunk 3]
```diff
@@ -0,0 +1,175 @@
+body 
+
+{
```
Comment (doubleedesign) [inline src/assets/styles.css:3]: CSS should be scoped to each component and placed in the `styled.ts` file in each component folder. Please use [the generator](https://redback-operations.github.io/redback-documentation/docs/web-mobile-app-dev/frontend/new-components) to create the proper files to start with.

File: src/routes/Root/React.jsx [hunk 4]
```diff
@@ -0,0 +1,1525 @@
+<div className="Group2" style={{width: 1919, height: 2663, position: 'relative'}}>
```
Comment (doubleedesign) [inline src/routes/Root/React.jsx:1]: What is this file for?

File: src/routes/Root/scripts.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
+document.addEventListener("DOMContentLoaded", function() {
```
Comment (doubleedesign) [inline src/routes/Root/scripts.js:1]: Please refactor all of this to belong in the relevant React components, using React conventions.

**Review by doubleedesign — CHANGES_REQUESTED** [review]

**Comment by doubleedesign** [comment]
I can see a lot of the build errors are related to whitespace, which is easy to fix by setting up your IDE to auto-format on save using the project's ESLint configuration. See [this doc](https://redback-operations.github.io/redback-documentation/docs/web-mobile-app-dev/frontend/submitting-work) for more information 🙂

**Comment by ben-AI-cybersec** [comment]
No attempt made to integrate reviewers requests. Code doesn't run, closing PR

**Pull request closed without merging**